sklearn/externals/_pilutil.py

```diff
@@ -40,7 +40,6 @@
 ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF
 THE POSSIBILITY OF SUCH DAMAGE.
 """
-from __future__ import division, print_function, absolute_import
```

Comment:
Please revert all changes under `sklearn/externals`: this is vendored code and it's better to keep it in its original state.